src/content/upgrades/merge/index.md

```diff
@@ -5,64 +5,236 @@ lang: en
+Blocks will instead be proposed by validating nodes that have ether staked for the right to participate in consensus. These upgrades set the stage for future scalability upgrades including data sharding.
 
 <ButtonLink to="/upgrades/beacon-chain/">
   The Beacon Chain
 </ButtonLink>
 
-### The Merge and the post-merge cleanup {#merge-and-post-merge-cleanup}
+### The Merge and the Shanghai upgrade {#merge-and-shanghai}
 
-Immediately after The Merge, some features such as withdrawing staked ETH, will not yet be supported. These are planned for a separate upgrade to follow shortly after The Merge.
+In order to simplify and maximize focus on a successful transition to proof-of-stake, The Merge upgrade will not include certain anticipated features such as the ability to withdraw staked ETH. The Shanghai upgrade is planned to follow The Merge, which will enable the ability for stakers to withdraw.
 
-Stay up-to-date with the [EF Research and Development Blog](https://blog.ethereum.org/category/research-and-development/). For those curious, learn more about [What Happens After The Merge](https://youtu.be/7ggwLccuN5s?t=101), presented by Vitalik at the April 2021 ETHGlobal event.
+Stay up-to-date with the [Shanghai upgrade planning issue on GitHub](https://github.com/ethereum/pm/issues/450), or the [EF Research and Development Blog](https://blog.ethereum.org/category/research-and-development/). For those curious, learn more about [What Happens After The Merge](https://youtu.be/7ggwLccuN5s?t=101), presented by Vitalik at the April 2021 ETHGlobal event.
 
-### The Merge and shard chains {#docking-and-shard-chains}
+### The Merge and data sharding {#merge-and-data-sharding}
 
-Originally, the plan was to work on shard chains before The Merge – to address scalability. However, with the boom of [layer 2 scaling solutions](/developers/docs/scaling/#layer-2-scaling), the priority has shifted to swapping proof-of-work to proof-of-stake via The Merge.
+Originally, the plan was to work on shard chains before The Merge in order to address scalability. However, with the boom of [layer 2 scaling solutions](/layer-2/), the priority has shifted to swapping proof-of-work to proof-of-stake via The Merge.
 
-This will be an ongoing assessment from the community as to the need for potentially multiple rounds of shard chains to allow for endless scalability.
+Plans for sharding are rapidly evolving, but given the rise and success of layer 2 technologies to scale transaction execution, sharding plans have shifted to finding the most optimal way to distribute the burden of storing compressed calldata from rollup contracts, allowing for exponential growth in network capacity. This would not be possible without first transitioning to proof-of-stake.
 
 <ButtonLink to="/upgrades/shard-chains/">
-  Shard chains
+  Data sharding
```

Comment:
Where can I read the discussion about using data sharding > just 'sharding'?

My gut take is that because we're still sharding Ethereum we should just say 'sharding'. Prefixing 'data' just adds more complexity for users.